src/components/NewCharacter/NewCharacter.js

```diff
@@ -99,22 +112,87 @@ const NewCharacter = ({ gameId, createCharacter }) => {
           onOptionSelected={option => setCharacterMorality(option)}
         />
       </div>
+      <div className={styles.abilitiesContainer}>
+        <PlusMinuser
+          label="Strength"
+          initialValue={strength}
+          min={characterRoll.min}
+          max={characterRoll.max}
+          canIncrement={canIncrementAbilities}
+          onChange={value => setStrength(value)}
+        />
+        <PlusMinuser
+          label="Dexterity"
+          initialValue={dexterity}
+          min={characterRoll.min}
+          max={characterRoll.max}
+          canIncrement={canIncrementAbilities}
+          onChange={value => setDexterity(value)}
+        />
+        <PlusMinuser
+          label="Constitution"
+          initialValue={constitution}
+          min={characterRoll.min}
+          max={characterRoll.max}
+          canIncrement={canIncrementAbilities}
+          onChange={value => setConstitution(value)}
+        />
+        <PlusMinuser
+          label="Intelligence"
+          initialValue={intelligence}
+          min={characterRoll.min}
+          max={characterRoll.max}
+          canIncrement={canIncrementAbilities}
+          onChange={value => setIntelligence(value)}
+        />
+        <PlusMinuser
+          label="Wisdom"
+          initialValue={wisdom}
+          min={characterRoll.min}
+          max={characterRoll.max}
+          canIncrement={canIncrementAbilities}
+          onChange={value => setWisdom(value)}
+        />
+        <PlusMinuser
+          label="Charisma"
+          initialValue={charisma}
+          min={characterRoll.min}
+          max={characterRoll.max}
+          canIncrement={canIncrementAbilities}
+          onChange={value => setCharisma(value)}
+        />
+      </div>
+      <div className={styles.abilitiesMetadata}>
+        Minimum: {characterRoll.min}
+        &nbsp; Maximum: {characterRoll.max}
+        &nbsp; Total: {characterRoll.total}
+        &nbsp; Remaining: {remainingPoints}
+      </div>
       <Button
-        className={styles.submit}
+        className={[styles.submit, styles.column1].join(' ')}
         text={'Create'}
         onClick={() => createCharacter({
           name: characterName,
           class: characterClass,
           race: characterRace,
           order: characterOrder,
           morality: characterMorality,
+          abilityScores: {
+            strength,
+            dexterity,
+            constitution,
+            intelligence,
+            wisdom,
+            charisma
+          },
           gameId
         })}
         disabled={!characterName ||
           !characterClass ||
           !characterRace ||
           !characterOrder ||
-          !characterMorality
+          !characterMorality ||
+          remainingPoints > 0
```

Comment:
Can we make this `remainingAbilityPoints` or alike, since we will have other types of points coming soon? (I think)